submissions/onyemeMichael/php-db/cookies.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+require_once("connection.php");
+
+echo "<h3> Customer Details </h3>";
+
+//Querying the database to select all information available in the table
+class QueryDb extends Connect {
+    public function customer() {
```

Comment:
I think `customers` is fine. Because you are not a returning a single customer but many